test/test_golog_app.cpp

```diff
@@ -0,0 +1,132 @@
+/***************************************************************************
+ *  test_golog_app.cpp - Test the golog application
+ *
+ *  Created:   Wed 11 Jan 14:11:00 CEST 2023
+ *  Copyright  2023  Daniel Swoboda <swoboda@kbsg.rwth-aachen.de>
+ *  SPDX-License-Identifier: LGPL-3.0-or-later
+ ****************************************************************************/
+
+#include "golog_app/app.h"
+
+#include <array>
+#include <catch2/catch_test_macros.hpp>
+#include <filesystem>
+
+TEST_CASE("Launch the main application", "[app]")
+{
+	const std::filesystem::path test_data_dir =
+	  std::filesystem::current_path() / "data" / "golog_app";
+	const std::filesystem::path plant_path            = test_data_dir / "robot_camera.gpp";
+	const std::filesystem::path spec_path             = test_data_dir / "robot_camera_spec.pbtxt";
+	const std::filesystem::path controller_dot_path   = test_data_dir / "controller.png";
+	const std::filesystem::path controller_proto_path = test_data_dir / "controller.pbtxt";
+	const std::filesystem::path tree_dot_graph        = test_data_dir / "tree.png";
+
+	std::vector<const char *> basic_parameters{"golog_app-exc",
+	                                           "--program",
+	                                           plant_path.c_str(),
+	                                           "--spec",
+	                                           spec_path.c_str(),
+	                                           "--k",
+	                                           "2",
```

Comment:
Why not use:
```suggestion
	                                           "1",
```
This speeds up the test (you also need to adapt the spec, see above)

Reply:
I need to investigate this, since it leads to no controller being synthesized anymore... not sure if it should be possible or not for this example.

Reply:
I guess that you also need to adapt the action durations accordingly, may that be the problem?

Reply:
I did that, but maybe I missed something, will try again to confirm.